### ISSUE TITLE
Python3 version?

### ISSUE THREAD

**Issue by lrosenman** [google/gmail-oauth2-tools#57]
Python2 is long deprecated, how can we use this script with Python3?

for starters:
```
✖1 ❯ oauth2 --generate_oauth2_token
  File "/Users/rosenmanl/bin/oauth2", line 297
    print 'Missing options: %s' % ' '.join(missing)
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
SyntaxError: Missing parentheses in call to 'print'. Did you mean print(...)?
```

**Comment by greatlordsecond** [comment]
https://github.com/google/gmail-oauth2-tools/compare/master...greatlordsecond:gmail-oauth2-tools:patch-1

**Comment by greatlordsecond** [comment]
Please check the above mentioned patch added compatibility for python3

**Comment by junyer** [comment]
Fixed in commit 944cffd66df38863eb0252a8096961335e418e70.
